java-manta-client-unshaded/src/test/java/com/joyent/manta/util/HttpClientBuilderShallowClonerTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2017, Joyent, Inc. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.joyent.manta.util;
+
+import com.joyent.manta.http.RequestIdInterceptor;
+import org.apache.commons.lang3.reflect.FieldUtils;
+import org.apache.http.HttpHost;
+import org.apache.http.HttpRequestInterceptor;
+import org.apache.http.HttpResponseInterceptor;
+import org.apache.http.client.HttpRequestRetryHandler;
+import org.apache.http.conn.HttpClientConnectionManager;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.LinkedList;
+
+import static org.mockito.Mockito.mock;
+
+@Test
+public class HttpClientBuilderShallowClonerTest {
```

Comment:
You may want to write a test or just try using the clone with a `SecurityManager` that denies reflection and see if the exception message output is useful when we get a report of it not working.